config/config_test.go

```diff
@@ -52,9 +51,24 @@ func TestNew(t *testing.T) {
 		t.Error("Backend for 'mem:' should be a MemoryBackend")
 	}
 
+	_, err = New("~/foobar")
+	if err != nil {
+		t.Fatalf("Failed here: %v", err)
+	}
+
 	_, err = New("c:\\foobar")
```

Comment:
According to https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats we could add some more test cases.